lib/phoenix/socket/transport.ex

```diff
@@ -388,6 +388,45 @@ defmodule Phoenix.Socket.Transport do
     end
   end
 
+  @doc false
```

Comment:
Why is it `@doc false`? :):

Reply:
its not because its private its because its undocumented :trollface:

Reply:
I can add a blurb